lib/ansible/modules/cloud/atomic/atomic_image.py

```diff
@@ -87,9 +103,38 @@ def core(module):
     image = module.params['name']
     state = module.params['state']
     started = module.params['started']
+    backend = module.params['backend']
     is_upgraded = False
 
     module.run_command_environ_update = dict(LANG='C', LC_ALL='C', LC_MESSAGES='C')
+    out = {}
+    err = {}
+    rc = 0
+
+    if backend:
```

Comment:
from what I can see the two branches are using different commands, e.g. /atomic pull/atomic run/atomic images delete/ and /atomic update/atomic run/atomic install/atomic uninstall/.  The only one in common seems to be "atomic run".  I am afraid it will be more difficult to merge the two cases